resources/views/success.blade.php

```diff
@@ -0,0 +1,63 @@
+
+@extends('layouts.app')
+
+@section('title', 'Course registered')
+
+@section('content')
+<div class="padded content">
+	<h1>Success! Your course was registered.</h1>
+	<div>
+		<h2>Overview:</h2>
+		<p>Course name: Cool course name</p>
+		<p>Teacher(s):</p>
+		<ul>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+			<li>Mr. Monday</li>
+		</ul>
+		<p>Course semester: Fall 2018</p>
+		<p>Course type: Client</p>
+		<p>Course schedule: Monday, Thursday</p>
+		<p>Starting time: 10:30 AM</p>
+		<p>Teams of: 4</p>
+		<p>Affiliated courses:</p>
+		<ul>
+			<li>Random</li>
+			<li>Random</li>
+			<li>Random</li>
+			<li>Random</li>
+		</ul>
+		<p>Your course key is ABCDEF.</p>
+	</div>
+</div>
+
+
+	{{-- <!-- Esto es lo que iría con la respuesta del backend -->
+	<div>
+		<h2>Overview:</h2>
+		<p>Course name: {{ $courseName }}</p>
+		<p>Teacher(s):</p>
+		<ul>
+		@foreach (teachers as teacher)
+			<li>{{ $teacher }}</li>
+		@endforeach
+		</ul>
+		<p>Course semester: {{ $courseSemester }}</p>
+		<p>Course type: {{ $courseType }}</p>
+		<p>Course schedule: {{ $courseSchedule }}</p>
+		<p>Starting time: {{ $courseHours }}</p>
+		<p>Teams of: {{ $teamsOf }}</p>
+		<p>Affiliated courses:</p>
+		<ul>
+		@foreach (affiliatedCourses as course)
+			<li>{{ $course }}</li>
+		@endforeach
+		</ul>
+		<p>Your course key is {{ $courseKey }}.</p>
+	</div> --}}
+
+@endsection
+
+@section('scripts')
+@endsection
```

Comment:
El view en gral está muy bien pero me parece que estaba mejor pensado en [el wireframe](https://github.com/ProyectoIntegrador2018/BlueMarket/issues/36#issuecomment-467266929). Sobre todo porque hay fields que tienen poquita información y podríamos inline them en un row con varios fields (como en el wireframe).

Reply:
Creo que así como lo tiene está bien? Particularmente para que sea consistente con mobile y que también todo se vea bien. Por cosas que estuve leyendo, no estoy segura de que inline sea lo mejor. [Check this out](https://uxplanet.org/10-rules-for-efficient-form-design-e13dc1fb0e03).

![image](https://user-images.githubusercontent.com/14140121/53619353-b185d180-3bb4-11e9-8d70-fad1ba104794.png)

Reply:
Maybe, pero también es cierto que hace que el form se vea _bastante_ más largo de lo que necesita ser y eso también es bad UX. Además de que genera confusión de por qué hay tanto real estate para un número. 

It's your call @melytc .

Reply:
Pues agree a los dos 😬
How about adjusting it if $(window).width() > 700? Like this:
<img width="1680" alt="screen shot 2019-03-01 at 12 54 26 am" src="https://user-images.githubusercontent.com/8518253/53626040-4e9f3500-3bca-11e9-834c-f8ae89d50eba.png">
<img width="1680" alt="screen shot 2019-03-01 at 12 50 07 am" src="https://user-images.githubusercontent.com/8518253/53626044-51018f00-3bca-11e9-88c2-59c0dc4aa0d9.png">

Reply:
Sii solo no lo hagas via JS. Semantic UI ya tiene clases para esto: https://semantic-ui.com/collections/grid.html#stackable